routines/populate.go

```diff
@@ -0,0 +1,50 @@
+package routines
+
+import (
+	"log"
+	"time"
+
+	"github.com/csivitu/bl0b/ctftime"
+	"github.com/csivitu/bl0b/db"
+)
+
+func populate(t time.Time) {
```

Comment:
It can't add the same event multiple times because ID is a primary key.

Reply:
I'm not sure if triggering database primary keys errors  intentionally is ideal, but it shouldn't be a an issue ig. If you're using postgres, it also has a `ON CONFLICT DO NOTHING` clause for insert statements, or even better `DO UPDATE` (which has the added benefit of  also updating event info) which would simulate how `upsert` works in mongo. You could look into those,  too, but mostly this can be resolved if its not a priority

Reply:
But I'm using `mysql`, is there an upsert option in that?

Reply:
I haven't used this before, but apparently there is 
https://stackoverflow.com/questions/6107752/how-to-perform-an-upsert-so-that-i-can-use-both-new-and-old-values-in-update-par

Reply:
:+1: I'll look into that

Reply:
All keys are updated on duplicate. @thebongy I tested this once, but just verify once anyway. Resolving this too.